plugins/generator-1.19.4/datapack-1.19.4/templates/structure/template_pool.json.ftl

```diff
@@ -8,10 +8,9 @@
       "element": {
         "element_type": "minecraft:single_pool_element",
         "location": "${modid}:${data.structure}",
-        "projection": "${data.projection}"
-        <#if data.ignoredBlocks?has_content>,
-        "processors": {
-          "processors": [
+        "projection": "${data.projection}",
+        "processors": [

```

Comment:
Is this flattening from

```
 "processors": {
          "processors": [
```

to

```
"processors": [
```

ok and tested? Is there any reason why this part was changed / what does it affect?

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/9f80d6e8-53e0-4c52-8bb3-a720c979d636)

I would keep this format as is

Reply:
✔️ Fixed